discography.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+	<title>Famous Person Website</title>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+
+<body>
+
+	<nav>
+	<a href="index.html">About</a> |
+	Discography
+	</nav>
+
+	<h1>Caetano Veloso</h1>
+
+	<h3>Discography</h3>
+
+	<table style="width:100%">
+  <tr>
```

Comment:
:hammer: This is a good example of why indentation is really important. Tables have a lot of nested content and it becomes difficult to read when the indentation is off. Excellent use of a table though!